Lib/gftools/encodings/GF Glyph Sets/GF-latin-core_unique-glyphs.nam

```diff
@@ -95,24 +96,18 @@
 0x007C | VERTICAL LINE
 0x007D } RIGHT CURLY BRACKET
 0x007E ~ TILDE
-0x00A0   NO-BREAK SPACE
 0x00A1 ¡ INVERTED EXCLAMATION MARK
 0x00A2 ¢ CENT SIGN
 0x00A3 £ POUND SIGN
-0x00A4 ¤ CURRENCY SIGN
-0x00A5 ¥ YEN SIGN
```

Comment:
I wouldn't remove the yen since it's heavily traded, https://en.wikipedia.org/wiki/Template:Most_traded_currencies. Please ignore if people represent it using other characters though.